modules/settings/settings.php

```diff
@@ -249,7 +252,7 @@ public function settings_validate( $new_options ) {
 	 */
 	public function helper_settings_validate_and_save() {
 
-		if ( ! isset( $_POST['action'], $_POST['_wpnonce'], $_POST['option_page'], $_POST['_wp_http_referer'], $_POST['vip_workflow_module_name'], $_POST['submit'] ) || ! is_admin() ) {
+		if ( ! isset( $_POST['action'], $_POST['_wpnonce'], $_POST['option_page'], $_POST['_wp_http_referer'], $_POST['submit'] ) || ! is_admin() ) {
```

Comment:
Removed `vip_workflow_module_name` from tests because we only have a single page that submits settings now.